BTVN10/main.js

```diff
@@ -0,0 +1,167 @@
+//b1:in bang nhan
+const getMultiply = (number) => {
+    console.log('In bang nhan:')
+    for(let i=1; i<=10; i++){
+        console.log(number, 'x', i, '=', number*i)
+    }
+}
+//b2:in cac so chan tu 1 den n
+const checkEven = (n) => {
+    if(n<1 || n>30) console.log('Nhap lai n')
+    else {
+        console.log('So chan tu 1 den', n, ':')
+        for(let i=1; i<=n; i++){
+            if(i%2==0) console.log(i)
```

Comment:
```suggestion
            if (i % 2 === 0) console.log(i)
```
Khi so sánh hãy dùng `===` thay vì dùng `==` để logic chặt chẽ hơn